dartdoc_test/example/main.dart

```diff
@@ -1,17 +1,3 @@
-// Copyright 2023 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
```

Comment:
any reason to strip this?

Reply:
We're generally required to have license headers on all files, see:
https://opensource.google/documentation/reference/releasing/preparing#license-headers

If there is a technical reason we don't want it on this particular file that's fine.

Reply:
I thought it would be enough to have the license headers in one file, but it seems that it needs to be included in all files. Thank you.